transport/src/main/java/io/netty/channel/internal/svm/NioEventLoopSubstitution.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.internal.svm;
+
+import java.util.Queue;
+import java.util.concurrent.LinkedBlockingDeque;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "io.netty.channel.nio.NioEventLoop")
+final class NioEventLoopSubstitution {
+
+    private NioEventLoopSubstitution() {
+    }
+
+    @Substitute
+    private static Queue<Runnable> newTaskQueue0(int maxPendingTasks) {
+        return new LinkedBlockingDeque<Runnable>(maxPendingTasks);
```

Comment:
can you explain why this is needed and why we can't use the default one ?

Reply:
That's something I took over from Quarkus, I guess JCTool's Queue isn't compatible with Graal, maybe @cescoffier  can chime in?

Reply:
also /cc @nitsanw @franz1981

Reply:
@cescoffier any idea ?

Reply:
Might not be required anymore. We need to check.